index.html

```diff
@@ -131,6 +131,13 @@ <h2 class="text-center" id="project-section-h2">In the News</h2>
                         </div>
                     </div>
                 </div>
+                <div class="row">
+                    <div class="col-md-6">
+                        <h3>Nerds brainstorm uses for open city data</h3>
+                        <p>hackMGM is continually using city data to with innovative tech projects</p>
```

Comment:
Looks like a grammar issue: "hackMGM is continually using city data to with..."

Also, I don't know that we've used city data yet so maybe:

"hackMGM plans to use city data with..."